languages/elixir/exercises/concept/anonymous-functions/.meta/example.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Secrets do
+
```

Comment:
I like the addition of the `secret_combine` function. The first three functions are still very similar, and a student would probably solve them by just copying/pasting the previous function. Is there are way to introduce a little bit of variation here?